wandb/sdk/wandb_init.py

```diff
@@ -382,9 +382,10 @@ def init(self):
             run._set_run_obj_offline(run_proto)
         else:
             ret = backend.interface.communicate_check_version()
-            message = ret.response.check_version_response.message
-            if message:
-                wandb.termlog(message)
+            if ret and ret.response and ret.response.check_version_response:
+                message = ret.response.check_version_response.message
```

Comment:
alternatively: try and except AttributeError

Reply:
i cleaned it up, just one "if" clause now